config/jest/CSSStub.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+module.exports = new Proxy({}, {
```

Comment:
Importing them just means the corresponding styles get attached, but there are no actual exports (at least not yet).